atlasdb-tests-shared/src/test/java/com/palantir/atlasdb/transaction/impl/SnapshotTransactionTest.java

```diff
@@ -2683,6 +2856,22 @@ private Transaction getSnapshotTransactionWith(
             LockImmutableTimestampResponse lockImmutableTimestampResponse,
             PreCommitCondition preCommitCondition,
             boolean validateLocksOnReads) {
+        return getSnapshotTransactionWith(
+                timelockService,
+                startTs,
+                lockImmutableTimestampResponse,
+                preCommitCondition,
+                validateLocksOnReads,
+                ImmutableMap.of(TABLE, Optional.of(ConflictHandler.RETRY_ON_WRITE_WRITE)));
+    }
+
+    private Transaction getSnapshotTransactionWith(
+            TimelockService timelockService,
+            Supplier<Long> startTs,
+            LockImmutableTimestampResponse lockImmutableTimestampResponse,
+            PreCommitCondition preCommitCondition,
+            boolean validateLocksOnReads,
+            Map<TableReference, Optional<ConflictHandler>> tableConflictHandlers) {
```

Comment:
I was considering to rewrite these static constructors to use an Immutable Builder class `TransactionOverrides`, but rejected the idea due to the increased refactoring effort. Thoughts?

Reply:
yeah let's not do this for now :)